doc/source/advanced.md

```diff
@@ -104,6 +106,19 @@ hub:
     }
 ```
 
+You can also specify `hub.extraConfig` as a dictionary, if you want to logically
+split your customizations. The code will be evaluated in alphabetical sorted
+order of the key.
+
+```yaml
+hub:
+  extraConfig:
```

Comment:
wouldn't there need to be a `|` after `extraConfig:`?

Reply:
Nope, because it's a dictionary. The | is a YAML feature that allows for multi-line strings, so we use it for the keys.